### DISCUSSION TITLE
Could I add a modified GeneIntegrationMatrix to arrow file?

### DISCUSSION THREAD

**Discussion by ChangliangWang** [GreenleafLab/ArchR#702] [Feature Requests]
Dear ArchR developers,

I have got the 'GeneIntegrationMatrix ' using 'getMatrixFromProject' function and I have modified the matrix. I am wondering how to add my modified 'GeneIntegrationMatrix' to the arrow file and use the modified 'GeneIntegrationMatrix'  to do downstream analysis.

Thanks very much for your help.

Best regards,
Changliang.

**Comment by rcorces** [comment]
This is not currently a feature in ArchR though it is planned in the future.

We've migrated feature requests to the Discussions platform in GitHub so I am moving this issue to the "Ideas" section there.

**Reply by mlin2017** [reply]
Hi @ChangliangWang and @rcorces ,

I am trying to modify an ArchR matrix as what was discussed here and then add the modified matrix back for downstream analysis. It sounds that this would be a new ArchR feature, and I would like to check if this feature has been released. If not yet, do you have any suggestions on possible solutions? @ChangliangWang How did you make it work?

Thanks,
Mao

**Reply by rcorces** [reply]
This is not yet available as a feature and I have not estimate for you as to when we will implement it.

**Reply by mlin2017** [reply]
Thank you for the response and updates.
